src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -0,0 +1,640 @@
+    }
+
+    private void SpeciesListMenuIndexChanged(int index)
+    {
+        var speciesName = speciesListMenu.Popup.GetItemText(index);
+        var species = speciesHistoryList[generationDisplayed].Values.First(p => p.FormattedName == speciesName);
+
+        if (species == speciesPreview.PreviewSpecies)
+            return;
+
+        UpdateSpeciesPreview(species);
+    }
+
+    private void UpdateSpeciesPreview(Species species)
+    {
+        speciesListMenu.Text = species.FormattedName;
+        speciesPreview.PreviewSpecies = species;
+        hexPreview.PreviewSpecies = species as MicrobeSpecies;
+        UpdateSpeciesDetail(species);
+    }
+
+    private void EvolutionaryTreeNodeSelected(int generation, uint id)
+    {
+        HistoryListMenuIndexChanged(generation);
+        UpdateSpeciesPreview(speciesHistoryList[generation][id]);
+    }
+
+    private void UpdateSpeciesDetail(Species species)
+    {
+        /*
+        speciesDetailsLabel.ExtendedBbcode = $"[b]Species:[/b]\n  {species.FormattedNameBbCode}:{species.ID}\n" +
+            $"[b]Generation:[/b]\n  {species.Generation}\n" +
+            $"[b]Population:[/b]\n  {species.Population}\n" +
+            $"[b]Colour:[/b]\n  #{species.Colour.ToHtml()}\n" +
+            $"[b]Behaviour[/b]\n  {species.Behaviour.Join(b => b.Key + ": " + b.Value, "\n  ")}\n";
+
+        switch (species)
+        {
+            case MicrobeSpecies microbeSpecies:
+            {
+                speciesDetailsLabel.ExtendedBbcode += $"[b]Stage:[/b]\n  Microbe\n" +
+                    $"[b]Membrane Type:[/b]\n  {microbeSpecies.MembraneType.Name}\n" +
+                    $"[b]Membrane Rigidity:[/b]\n  {microbeSpecies.MembraneRigidity}\n" +
+                    $"[b]Base Speed:[/b]\n  {microbeSpecies.BaseSpeed}\n" +
+                    $"[b]Base Rotation Speed[/b]\n  {microbeSpecies.BaseRotationSpeed}\n" +
+                    $"[b]Base Hex Size[/b]\n  {microbeSpecies.BaseHexSize}";
+                break;
+            }
+        }
+        */
+
+        speciesDetailsLabel.ExtendedBbcode = string.Format(CultureInfo.CurrentCulture,
+            TranslationServer.Translate("SPECIES_DETAIL_TEXT"), species.FormattedNameBbCode, species.ID,
+            species.Generation, species.Population, species.Colour.ToHtml(),
+            species.Behaviour.Join(b => b.Key + ": " + b.Value, "\n  "));
+
+        switch (species)
+        {
+            case MicrobeSpecies microbeSpecies:
+            {
+                speciesDetailsLabel.ExtendedBbcode += string.Format(CultureInfo.CurrentCulture,
+                    TranslationServer.Translate("MICROBE_SPECIES_DETAIL_TEXT"), microbeSpecies.MembraneType.Name,
+                    microbeSpecies.MembraneRigidity, microbeSpecies.BaseSpeed, microbeSpecies.BaseRotationSpeed,
+                    microbeSpecies.BaseHexSize);
+                break;
+            }
+        }
+    }
+
+    private void PlayWithCurrentSettingPressed()
```

Comment:
Ah, you implemented the enter the game feature, I guess then my previous comment about adding a TODO about the player should be ignored. Instead could you add a comment on the luca variable that it'd be nice to unmark the species as the player species while in the tool, because being the player species gives it an advantage, and probably stops it from evolving?